recipes/remove.rb

```diff
@@ -33,3 +33,19 @@
 file '/usr/libexec/ucarp/ucarp' do
   action :delete
 end
+
+ucarp_databag = begin
+                  data_bag_item('ucarp', node['ucarp']['data_bag']['cluster'])
+                rescue
+                  nil
+                end
+
+unless ucarp_databag.nil?
+  file "/etc/ucarp/vip-#{ucarp_databag['vip_id']}.pwd" do
+    action :delete
+  end
+
+  template "/etc/ucarp/vip-#{ucarp_databag['vip_id']}.conf" do
+    action :delete
+  end
```

Comment:
Instead of doing it this way, I recommend you use the ``edit_resource`` and change the action to delete for both. You may also need to move this above the package and service removal.

Reply:
I've switched to using `edit_resource`.